src/cls/IPM/Storage/Module.cls

```diff
@@ -1434,9 +1434,26 @@ Method %Evaluate(pAttrValue As %String, ByRef pParams) As %String [ Internal ]
 	Set name = ..Name
 	Set attrValue = ##class(%Regex.Matcher).%New("(?i)\{name\}", attrValue).ReplaceAll($Replace(name,"\","\\"))
 
+	Set regex = ##class(%Regex.Matcher).%New("#\{([^}]+)\}", tAttrValue)
+	While regex.Locate() {
+		Set expr = regex.Group(1)
+		Set value = ..%EvaluateExpression(expr)
+		Set $Extract(attrValue, regex.Start, regex.End - 1) = value
+		Set regex.Text = attrValue
+	}
+
 	Return attrValue
 }
 
+Method %EvaluateExpression(pExpr) As %String [ Internal ]
```

Comment:
Should we support evaluation of arbitrary expressions from a security perspective? I thought we just do string interpolation of the expressions

Reply:
From a security perspective, IPM already supports evaluation of arbitrary expressions in a much less transparent manner via Invoke elements. There are times where this kind of approach is necessary.

Reply:
Recent motivating example: needing to be able to reference ^%SYS("WebServer","URLPrefix") in a deployment URL for a React application, so that it'll work on any local IRIS instance using a real webserver. This is something that could ultimately have its own alias for string interpolation, but a package can't necessarily wait for an IPM enhancement to make that happen.

Reply:
@isc-tleavitt  Note that using that value isn't helpful or newer versions of IRIS with NoPWS so should likely be provided when installing the module